server/data/visitSchedulerApiClient.ts

```diff
@@ -54,4 +54,10 @@ export default class VisitSchedulerApiClient {
       }).toString(),
     })
   }
+
+  async getSingleSessionTemplate(reference: string): Promise<SessionTemplate> {
```

Comment:
Just `getSessionTemplate` would be more consistent with how we've handled calls for single / multiple elsewhere. Don't have a strong preference; I guess this is more descriptive 🤷‍♂️.

Reply:
wasn't sure on what to go for, for this one - with the names being so similar it (template vs templates) it was confusing to read and see what was going on. 
Happy to use either though.

Reply:
Yeah, happy to stick with more descriptive one - no strong preference